modules/gds_printers/manifests/init.pp

```diff
@@ -0,0 +1,20 @@
+# Configure printers for Aviation House.
+#
+# You will need to override `ldap_username` if your LDAP username differs
+# from your local login username.
```

Comment:
Actually this comment is wrong - what you mean is "if your LDAP username differs from your GitHub username", since that's what `$boxen_user` will be set to.